plugins/org.jboss.tools.arquillian.reddeer/src/org/jboss/tools/arquillian/ui/bot/reddeer/configurations/RunConfigurationsDialog.java

```diff
@@ -1,26 +1,37 @@
+/*******************************************************************************
```

Comment:
RedDeer has LaunchConfigurationDialog and RunConfigurationDialog. It may be worth a while to use this instead. But this is not blocking this PR.